sky/packages/sky/lib/src/animation/curves.dart

```diff
@@ -153,38 +153,43 @@ class ElasticInOutCurve implements Curve {
   }
 }
 
-/// A linear animation curve
-const Linear linear = const Linear();
+/// A collection of common animation curves.
+class Curves {
+  Curves._();
 
-/// A cubic animation curve that speeds up quickly and ends slowly
-const Cubic ease = const Cubic(0.25, 0.1, 0.25, 1.0);
+  /// A linear animation curve
+  static const Linear linear = const Linear();
 
-/// A cubic animation curve that starts slowly and ends quickly
-const Cubic easeIn = const Cubic(0.42, 0.0, 1.0, 1.0);
+  /// A cubic animation curve that speeds up quickly and ends slowly
+  static const Cubic ease = const Cubic(0.25, 0.1, 0.25, 1.0);
 
-/// A cubic animation curve that starts quickly and ends slowly
-const Cubic easeOut = const Cubic(0.0, 0.0, 0.58, 1.0);
+  /// A cubic animation curve that starts slowly and ends quickly
+  static const Cubic easeIn = const Cubic(0.42, 0.0, 1.0, 1.0);
 
-/// A cubic animation curve that starts slowly, speeds up, and then and ends slowly
-const Cubic easeInOut = const Cubic(0.42, 0.0, 0.58, 1.0);
+  /// A cubic animation curve that starts quickly and ends slowly
```

Comment:
You can experiment with them here:

https://developer.mozilla.org/en-US/docs/Web/CSS/transition-timing-function

easeOut is symmetric.  ease gets closer to the destination faster.